stable/artifactory-ha/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- unguiculus
+- prydonius
+- mattfarina
```

Comment:
While appreciate being added to this approvers file, as a repo maintainer I do not need to be in here. This applies for any repo maintainers. Can you remove us and only have the chart maintainers listed?

Reply:
@mattfarina sure